src/jest-types.ts

```diff
@@ -1,12 +1,13 @@
-export type TransformOptions = {
+import { TransformOptions as BabelTransformOpts } from 'babel-core';
+export interface TransformOptions {
   instrument: boolean;
-};
+}
```

Comment:
Converted it from a type to a interface, and generally interfaces don't have semicolons at the end.